hardhat.config.ts

```diff
@@ -67,4 +67,4 @@ export default {
   mocha: {
     timeout: 1000000000000
   }
-}
+}
```

Comment:
Strange, it was made by the prettier?

Reply:
No, because during the merge was kept [hardhat.config.ts](https://github.com/matter-labs/era-permit2/blob/0x000000000022D473030F116dDEE9F6B43aC78BA3-zksync-era/hardhat.config.ts) from matter-labs repo for solve conflict.

Reply:
So, please rerun prettier, or fix it manually